setup.py

```diff
@@ -1,5 +1,73 @@
-#!/usr/bin/env python
-"""See https://github.com/allenai/allennlp/blob/master/setup.py to create a package for PyPI."""
-from setuptools import setup
+from setuptools import find_packages, setup
 
-setup()
+# PEP0440 compatible formatted version, see:
+# https://www.python.org/dev/peps/pep-0440/
+#
+# release markers:
+#   X.Y
+#   X.Y.Z   # For bugfix releases
+#
+# pre-release markers:
+#   X.YaN   # Alpha release
+#   X.YbN   # Beta release
+#   X.YrcN  # Release Candidate
+#   X.Y     # Final release
+
+# version.py defines the VERSION and VERSION_SHORT variables.
+# We use exec here so we don't import allennlp whilst setting up.
+
+VERSION = {}  # type: ignore
+with open("allennlp_server/version.py", "r") as version_file:
+    exec(version_file.read(), VERSION)
+
+setup(
+    name="allennlp-server",
+    version=VERSION["VERSION"],
+    description="Simple demo server for AllenNLP models and training config builder.",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Framework :: Flake8",
+        "Framework :: Flask",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Artificial Intelligence",
+        "Topic :: Scientific/Engineering :: Visualization",
```

Comment:
Not sure if this is applicable. 
Probably Dirk can confirm.

Reply:
I don't think we can say this. This doesn't really visualize the output (though we keep talking about doing that at some point ...).

Reply:
Cool. Will remove this.